### PR TITLE
refactor: input styling

### PR DESCRIPTION
**Pull request by yndajas**
Changes:
- placeholder text colour
- spacing within numbered questions with multiple inputs, including dynamic panels - where dynamic panels were not used, e.g. on the location page, the model has been updated to wrap these in panels (helps with spacing fixes)
- file upload styling
- input font sizes
- question font sizes on page two, question two (check for conflicts with #46)
- replace class lists with mixins

### CODE REVIEW

File: app/assets/stylesheets/base/_surveyjs.scss [hunk 1]
```diff
@@ -190,4 +247,111 @@ $radio-checkbox-width-height: rem-calc(39);
...
+    }
+
+    // delete button icon
+    &__remove-svg {
+      fill: $danger-dark;
+
+      display: flex;
+
+      position: static;
+    }
```
Comment (RainbowGiantSquid) [inline app/assets/stylesheets/base/_surveyjs.scss:336]: This looks great but when a graphics file is uploaded and a pre-view is available the close button gets pushed bottm right :(
image

File: app/assets/stylesheets/base/_surveyjs.scss [hunk 2]
```diff
@@ -190,4 +247,111 @@ $radio-checkbox-width-height: rem-calc(39);
...
+        order: 2;
+      }
+    }
+
+    // delete button icon
+    &__remove-svg {
+      fill: $danger-dark;
+
+      display: flex;
+
```
Comment (RainbowGiantSquid) [inline app/assets/stylesheets/base/_surveyjs.scss:334]: whitespace

**Review by RainbowGiantSquid — CHANGES_REQUESTED** [review]
Thanks Ynda! All looks good to me just a couple of small things. Love all the mixins, very organised!

File: app/assets/stylesheets/base/_surveyjs.scss (hunk 1 above)
Comment (yndajas) [reply app/assets/stylesheets/base/_surveyjs.scss:336]: Removed preview from second file upload

**Pull request merged**